src/components/configdrop/ConfigDrop.test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as enzyme from "enzyme";
+import { DropzoneArea } from 'material-ui-dropzone';
+import * as React from "react";
+import { ClientConfig } from '../../models/client_config';
+import { ConfigDrop } from "./ConfigDrop";
+
+describe("ConfigDrop", () => {
+    const verifyConfig = (configObj: ClientConfig) => {
+      if (configObj !== undefined) {
+        expect(configObj.user).toBe("some.user");
+        expect(configObj.api_key).toBe("aczq3eG0gAgEV9uGZN5gCos1utnZPXIg");
+        expect(configObj.api_url).toBe("https://evergreen.mongodb.com/api");
+        expect(configObj.ui_url).toBe("https://evergreen.mongodb.com");
+      }
+     }
+    const wrapper = enzyme.mount(<ConfigDrop updateClientConfig={verifyConfig}/>);
+
+    it("matches snapshot", () => { 
+      // note: using enzyme.mount here (instead of enzyme.shallow) causes the test suite to hang, so this test needs its own shallow wrapper
+      const shallowWrapper = enzyme.shallow(<ConfigDrop updateClientConfig={verifyConfig}/>);
+      expect(shallowWrapper).toMatchSnapshot();
+    })
+
+    it("check config state object is not updated with an empty file", () => {
+      // check that the config object in state is null initially
```

Comment:
There are differing opinions on this, but I personally think comments should only be used to explain things that may not be obvious. Could you remove the comments that explain the same thing that reading the code would?